superset-frontend/packages/superset-ui-core/src/number-format/factories/createNetworkNumberFormatter.ts

```diff
@@ -0,0 +1,156 @@
+// @ts-nocheck
+
+import { format as d3Format } from 'd3-format';
+import NumberFormatter from '../NumberFormatter';
+import NumberFormats from '../NumberFormats';
+
+const float2PointFormatter = d3Format(`.2~f`);
+const float4PointFormatter = d3Format(`.4~f`);
+
+const bytesSILabels = ['Bytes', 'kB', 'MB', 'GB', 'TB', 'PB', 'EB', 'ZB'];
+const bytesIECLabels = [
+  'Bytes',
+  'KiB',
+  'MiB',
+  'GiB',
+  'TiB',
+  'PiB',
+  'EiB',
+  'ZiB',
+];
```

Comment:
Is there overlap here with the code in `createMemoryFormatter.ts`? I think ideally we want to avoid duplication and double down on the same terms. Some quick observations:
- Here we're talking about `Bytes`, while in memory formatter we have `B`
- Here we're only going to `ZiB`/`ZB`, while in memory formatter we're going all the way to `YiB` and for SI to `QB`.